src/main/scala/com/raquo/airstream/eventstream/EventStream.scala

```diff
@@ -36,6 +36,8 @@ trait EventStream[+A] extends LazyObservable[A] {
     new FilterEventStream(this, passes)
   }
 
+  def filterNot(passes: A => Boolean): EventStream[A] = filter(passes.andThen(!_))
```

Comment:
We'll instantiate one less Function if we just do `filter(!passes(_))`.

This might be a moot point after `fullOpt` but it can't hurt

Reply:
Oh also, `passes` is no longer the right name, since its result indicates the opposite of passing.

Perhaps `skip`? Or maybe `predicate` if that doesn't feel right. I like having semantic names for such params, not just `f`...

Reply:
renamed to `predicate`

Reply:
also added a couple of tests (for map-flatten)